_gsocprojects/2020/project_ATLAS.md

```diff
@@ -0,0 +1,9 @@
+---
+project: ATLAS
+layout: default
+logo: ATLAS-Logo-Ref-RGB.png
```

Comment:
The logo has to be named "ATLAS-logo.png"

Reply:
Done.